lightning/tests/config/pd.toml

```diff
@@ -1,6 +1,3 @@
-lease = 360
```

Comment:
Do we need to update BR test config?

Reply:
https://github.com/pingcap/tidb/pull/55498/commits/4ed8bf7d95a7a23df8706a68219644c1651c5ae2

I have confirmed with @YuJuncen . @YuJuncen  PTAL